### ISSUE TITLE
Possibly split Woodsman into Carpenter and Hunter

### ISSUE THREAD

**Issue by gjpeebles** [Tirocupidus/TheExiledRPOverhaul#17]
The Woodsman has a lot of stuff.

**Comment by Takeyo-Efthulc** [comment]
I would agree with this. Another thing I'd like to see for Woodsman is putting both kinds of leather in T1, because it's just leather. There's nothing special about it being thicker. It also would make more sense if the rawhide bindings were in T1 too, since there's really no point in having both kinds of bindings when they're in the same tier. Another thing, for realism's sake, would be to put all the trophies at T2+. Mounting trophies is hard. Tanning hides is not.

On a similar note, I don't think that potters should have to also be builders, but that's a good point for another thread.
